src/bin/ion/commands/beta/count.rs

```diff
@@ -0,0 +1,52 @@
+use crate::commands::CommandConfig;
+use anyhow::{Context, Result};
+use clap::{App, Arg, ArgMatches};
+use ion_rs::*;
+use std::fs::File;
+use std::io::{stdin, BufReader, StdinLock};
+
+pub fn app() -> CommandConfig {
+    App::new("count")
+        .about("Prints the number of top-level values found in the input stream.")
```

Comment:
🗺️ I've used this program locally for debugging/sanity checking and thought it would be nice to include it in the CLI. It's under the `beta` subcommand, so we can change it as needed.